packages/next-remoter/src/composable/useRouteBasedTools.ts

```
@@ -0,0 +1,169 @@
+      if (isActiveRoute) {
+        activeToolIds.add(toolName)
+      }
+    })
+
+    // 2. 更新 ignoreToolnames：只保留非激活工具
+    const ignore = new Set<string>(agent.ignoreToolnames as string[])
+    // 将所有路由工具先加入忽略集合，然后移除当前激活的
+    routeMap.forEach((_route, toolName) => {
+      ignore.add(toolName)
+    })
+    activeToolIds.forEach((id) => ignore.delete(id))
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)